src/commands/image-push-to-registries.yaml

```diff
@@ -0,0 +1,136 @@
+parameters:
+  image-sha256_envvar:
+    type: "env_var_name"
+  image:
+    type: "string"
+  tag_envvar:
+    type: "env_var_name"
+  tag-latest-branch:
+    type: "string"
+  tag-suffix:
+    type: "string"
+    default: ""
+  build-context:
+    type: "string"
+    default: "."
+  dockerfile:
+    type: "string"
+    default: "./Dockerfile"
+  registries-data:
+    default: ""
+    description: |
+      A string that defines configuration for registries
+      Each line describes one registry using the following format
+
+      visibility registry username_envvar password_envvar push_dev_image
+
+      where
+
+      access is one of ("public" "private" "public/private")
+      registry is a string containing registry url
+      username_envvar is a string containing the name of an env var with a username
+      password_envvar is as string containing the name of an env var with a password
+      push_dev is boolean, when true, dev images are uploaded to registries
+    type: string
+  force-public:
+    type: boolean
+    default: false
+steps:
+  - run:
+      name: Login to registries
+      command: |
+        set -x
+
+        if ! [ -z "<<parameters.registries-data>>" ]; then
```

Comment:
Nitpick, but since this will run for hundreds of times a day... I think we can use bash checks `[[` vs invoking external test binary `[` - should be more efficient.

Reply:
Should be fixed